script/state.py

```diff
@@ -13,6 +15,10 @@
 states_data = cur.fetchall()
 conn.close()
 
+if len(states_data)==0:
+    print("No data fetched from database ")
```

Comment:
Use logging here instead of print statements, check https://docs.python.org/3/howto/logging.html and the simple example mentioned in this link.